src/current/v24.3/cockroachdb-in-comparison.md

```diff
@@ -282,14 +282,12 @@ This page shows you how the key features of CockroachDB stack up against other d
     <td class="comparison-chart__column-one">
       <span class="support " data-dbs='["MySQL", "PostgreSQL", "Oracle","AWS Aurora"]'>No</span>
       <span class="support " data-dbs='[ "Cassandra"]'>Yes, object level</span>
-      <span class="support " data-dbs='["MongoDB", "Spanner"]'>Yes</span>
-      <span class="support " data-dbs='["Yugabyte"]'>No</span>
+      <span class="support " data-dbs='["MongoDB", "Spanner", "Yugabyte"]'>Yes</span>
     </td>
     <td class="comparison-chart__column-two">
           <span class="support " data-dbs='["MySQL", "PostgreSQL", "Oracle","AWS Aurora","MongoDB"]'>No</span>
       <span class="support " data-dbs='[ "Cassandra"]'>Yes, object level</span>
-      <span class="support " data-dbs='["Spanner"]'>Yes</span>
-      <span class="support " data-dbs='["Yugabyte"]'>No</span>
+      <span class="support " data-dbs='["Spanner", "Yugabyte"]'>Yes</span>
```

Comment:
Yugabyte also supports row-level geo-partitioning: https://docs.yugabyte.com/preview/explore/multi-region-deployments/row-level-geo-partitioning

I'd just change both Yugabyte's and CockroachDB's values to "Yes".